### PR TITLE
Fix the welter boss

### LINKED ISSUE #714: The Welter (egg spawn and spawn of the welter)
### Priority

Low

### Area

- [ ] Datapack
- [ ] Source
- [ ] Map
- [X] Other

### What happened?

Does anyone has a welter that works correctly?

image

Currently it does not spawn the eggs nor his spawns which makes the fight....kind of easy

### What OS are you seeing the problem on?

Ubuntu 20.04, Ubuntu 22.04

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by murilo09**
# Description

Fix The Welter Boss

### **Actual**

There is no the welter summons and other small errors in the welter mechanic

### **Expected**

Add The Welter Summons and fix in the amount of eggs summons

## Fixes

opentibiabr/otservbr-global#714 

## Type of change

Please delete options that are not relevant.

  - [X] Bug fix (non-breaking change which fixes an issue)
  - [X] New feature (non-breaking change which adds functionality)

## Checklist

  - [X] My code follows the style guidelines of this project
  - [X] I have performed a self-review of my own code
  - [X] I checked the PR checks reports
  - [ ] I have commented my code, particularly in hard-to-understand areas
  - [ ] I have made corresponding changes to the documentation
  - [X] My changes generate no new warnings
  - [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by murilo09** [comment]
Changed to draft while investigating #766

File: data/monster/raids/the_welter.lua [hunk 1]
```diff
@@ -149,4 +149,4 @@ end
 mType.onSay = function(monster, creature, type, message)
 end
 
-mType:register(monster)
+mType:register(monster)
```
Comment (majestyotbr) [inline data/monster/raids/the_welter.lua:152]: Add new line

**Comment by majestyotbr** [comment]
Tested and working.

**Review by majestyotbr — COMMENTED** [review]
It's working and fixed the mentioned issue.

**Pull request merged**